script.js

```diff
@@ -0,0 +1,656 @@
+        const key = arrKeys[i][j];
+        let classesStr = `key ${key}`.toLowerCase();
+
+        if (arrFuncKeys.includes(key)) classesStr = `${classesStr} key--function`;
+
+        const keyElm = this.createNewElement({ tag: 'button', tagClass: classesStr, tagText: '' });
+        keyElm.setAttribute('data-key-code', key);
+        keysRow.append(keyElm);
+      }
+      keyBox.append(keysRow);
+    }
+    return keyBox;
+  },
+
+
+  // method for make keyboard language layout, argument:('en' or 'ru')
+  setLanguageLayout(languageToggle) {
+    const arrKeys = this.elements.keyBox.querySelectorAll('.key');
+    for (let i = 0; i < arrKeys.length; i += 1) {
+      const elem = arrKeys[i];
+      const keySymbol = this.language[languageToggle].lowerCase[elem.dataset.keyCode];
+      elem.innerHTML = keySymbol;
+    }
+    this.inputAria.languageToggle = languageToggle;
+    localStorage.setItem('virtualKeyboardLang', languageToggle);
+  },
+
+
+  // method set uppercase keyboard when downKey Shift, argument:(not)
+  setShiftDownLayout() {
+    const arrKeys = this.elements.keyBox.querySelectorAll('.key:not(.key--function)');
+    for (let i = 0; i < arrKeys.length; i += 1) {
+      const elem = arrKeys[i];
+      const keySymbol = this.language[this.inputAria.languageToggle].shift[elem.dataset.keyCode];
+      elem.innerHTML = keySymbol;
+    }
+  },
+
+
+  // method set lowercase keyboard when upKey Shift, argument:(not)
+  setShiftUpLayout() {
+    const arrKeys = this.elements.keyBox.querySelectorAll('.key');
+    for (let i = 0; i < arrKeys.length; i += 1) {
+      const elem = arrKeys[i];
+      const keySymbol = this.language[this.inputAria.languageToggle]
+        .lowerCase[elem.dataset.keyCode];
+      elem.innerHTML = keySymbol;
+    }
+  },
+
+
+  // method switch lowercase keyboard when upKey CapsLock, argument:(not)
+  setCapsLock() {
+    const arrKeys = this.elements.keyBox.querySelectorAll('.key:not(.key--function)');
+
+    for (let i = 0; i < arrKeys.length; i += 1) {
+      const elem = arrKeys[i];
+      const keySymbol = elem.innerHTML.toUpperCase();
+      elem.innerHTML = keySymbol;
+    }
+  },
+
+
+  // method switch Language Layout for keyboard, argument:(not)
+  switchLanguageLayout() {
+    if (this.inputAria.languageToggle === 'en') {
+      this.inputAria.languageToggle = 'ru';
+    } else if (this.inputAria.languageToggle === 'ru') {
+      this.inputAria.languageToggle = 'en';
+    }
+    this.setLanguageLayout(this.inputAria.languageToggle);
+  },
+
+
+  // method type from screen keys
+  type(char) {
+    const text = this.elements.textAria.value;
+    const start = this.elements.textAria.selectionStart;
+    const end = this.elements.textAria.selectionEnd;
+
+    this.elements.textAria.value = `${text.substring(0, start)}${char}${text.substring(end)}`;
+    this.elements.textAria.selectionStart = start + 1;
+    this.elements.textAria.selectionEnd = start + 1;
+  },
+
+
+  // method type Backspace key
```

Comment:
such comments are redundant, you have quite descriptive names

Reply:
Done